tests/run-make/x86_64-fortanix-unknown-sgx-lvi/rmake.rs

```diff
@@ -0,0 +1,92 @@
+// This security test checks that the disassembled form of certain symbols
+// is "hardened" - that means, the assembly instructions match a pattern that shows
+// lack of vulnerability to a Load Value Injection attack.
```

Comment:
Question (to sgx maintainers): is this statement accurate, or is the test demonstrating a particular kind of hardening or mitigations for a particular kind of LVI attack?

Reply:
(Also, as mentioned in another review comment, do we know which particular kinds here, if applicable?)

Reply:
The tests don't differentiate between different types of LVI attacks